src/test/http_rest_api_handler_test.cpp

```diff
@@ -625,6 +626,82 @@ TEST_F(ConfigReload, StartWith1DummyThenReloadToAddPipeline) {
     EXPECT_EQ(status, ovms::StatusCode::OK_RELOADED);
 }
 
+TEST_F(ConfigReload, StartWith1DummyThenReloadToMediapipe) {
+    ovms::Server& ovmsServer = ovms::Server::instance();
+    TestHelper1 t(*this, configWith1Dummy);
+    auto handler = ovms::HttpRestApiHandler(ovmsServer, 10);
+
+    std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
What we expect to happen in  1 sec? Some kind of initial load ?
What will happen if machine will be overloaded and 1sec will be not enough ?

Reply:
Standard timeout for models configs reload used in tests.